### PR TITLE
Bump version to 1.2.0

### PR DESCRIPTION
**Pull request by guarin**

### CODE REVIEW

File: lightly/__init__.py
Comment (philippmwirth) [inline lightly/__init__.py:78]: We should decide for the whole project whether we want to do double quotes or single quotes.

**Comment by philippmwirth** [comment]
Can we make sure all the issues which need to be done are done before merging this? Are there any open questions?

**Review by MalteEbner — APPROVED** [review]

**Pull request merged**
